nntrainer/tensor/tensor.h

```diff
@@ -23,7 +23,9 @@
 
 #include <cstddef>
 
-#include <blas_interface.h>
+#include <char_tensor.h>
+#include <cpu_backend.h>
+#include <float_tensor.h>
```

Comment:
no need to include char_tensor.h and float_tensor.h!

```suggestion
#include <cpu_backend.h>
```